test/integration/targets/sts_assume_role/tasks/main.yml

```diff
@@ -0,0 +1,382 @@
+---
+# tasks file for sts_assume_role
+
+- block:
+
+    # ============================================================
+    # TODO create simple ansible sts_get_caller_identity module
+    - blockinfile:
+        path: /tmp/sts.py
```

Comment:
Use `{{ output_dir }}/sts.py` instead. This will keep the temporary file private to this test run.

Reply:
Fixed